.github/workflows/docker-release.yml

```diff
@@ -13,6 +13,8 @@ on:
     tags:
       # Release candidate tags look like: v1.11.0-rc1
       - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
```

Comment:
But what's the point of running it on RCs? Should we remove those?

Reply:
Agree, I validated that I can manually kick this workflow. No need to run it on RC: https://github.com/pytorch/pytorch/actions/runs/7479615393/job/20357266572